### DISCUSSION TITLE
Workshop.md mentions using an existing example for our own models

### DISCUSSION THREAD

**Discussion by javac9** [openmdx/openmdx-example#10] [Q&A]
In `workshop.md` it is said:
`
NOTE: Using EMF and Papyrus requires some experience. In order to create a model for a new project, it is a good idea to start with the model file from the Workshop project. In a second step adapt the model package names and then add and remove classes and associations to your needs.
`

What is the recommended way of renaming the package `org.openmdx.example.workshop1` to something else by which all or the majority of dependent stuff would be also adjusted?

I tried manually renaming the package in Model explorer but then I noticed the new UML file was created under EMF.

If I'm not asking for too much, please also shed some light onto the interconnection between EMF and Papyrus directories, so we can better understand their relationship.

**Comment by wfro64** [comment]
The simplest way to rename the model `workshop1` is to open the model files (./src/model/emf/org.openmdx.example.workshop1.uml, ./src/model/papyrus/models.notation, ...) in a text editor and do the renaming there.

**Reply by javac9** [reply]
I renamed occurrences of `org.openmdx.example.workshop1` to `com.example.app1`. After `gradle clean` and `gradle assemble` I can see the imports in `TestWorkshop_1` were also renamed are not correct anymore.

The reason is: `./build/generated/sources/java/main` still contains the old package hierarchy: 
```
-| org
  -| openmdx
    -| example
      -| workshop1
        -| jmi1
          -| Skill
```
Is that expected or I missed something?
